frontends/web/src/components/devices/bitbox02/setdevicename.jsx

```diff
@@ -87,10 +87,8 @@ export class SetDeviceName extends Component {
                             <div className="columnsContainer half">
                                 <div className="columns half">
                                     <div className="column">
-                                        <Input
-                                            label="Current Device Name"
-                                            disabled={true}
-                                            value={name} />
+                                        <label>Current Device Name</label>
```

Comment:
please make a habit of putting new text in `t('...')`, as it's harder to find all the un-locized things after